guide/popular-topics/canvas.md

```diff
@@ -140,17 +136,13 @@ client.on('interactionCreate', async interaction => {
 
 A bit plain, right? Fear not, for you have a bit more to do until you reach completion. Since this guide page's goal is focused more on actual code than design, let's place a basic square-shaped avatar for now on the left side of the image. In the interest of coverage, you will also make it a circle afterward.
 
-```js {5-9}
-const { request } = require('undici');
-
+```js {3-5}
 client.on('interactionCreate', async interaction => {
 	// ...
 	context.strokeRect(0, 0, canvas.width, canvas.height);
 
-	const { body } = await request(interaction.user.displayAvatarURL({ format: 'jpg' }));
-	const avatar = new Canvas.Image();
-	avatar.src = Buffer.from(await body.arrayBuffer());
-
+	const avatar = await Canvas.loadImage(interaction.user.displayAvatarURL({ format: 'jpg' }));
```

Comment:
This is actually a downgrade performance-wise, since `@napi-rs/canvas` uses the native `https` module under the hood
It might be a good idea to keep `undici.request()` and only update the `new Canvas.Image()` part

Reply:
@almeidx I've made some changes, is it good now?

Reply:
Yes